doc/api/console.markdown

```diff
@@ -48,6 +48,10 @@ is used on each argument.  See [util.format()][] for more information.
 
 Same as `console.log`.
 
+### console.debug([data][, ...])
```

Comment:
Then we might want to change the complete doc for console. Because the signature to methods like log are also not matching with spec we are following https://github.com/nodejs/node/blob/master/doc/api/console.markdown#consolelogdata- Do you think I should change those them as well?